shapeworks_cloud/core/models.py

```diff
@@ -5,14 +5,48 @@
 from .metadata import METADATA_FIELDS, generate_filename
 
 
-class Dataset(TimeStampedModel, models.Model):
+class FormattedSizeMixin:
+    @property
+    def formatted_size(self, base=1024, unit='B'):
+        size = self.size
+        if size < base:
+            return f'{size} {unit}'
+        units = ['', 'K', 'M', 'G', 'T']
+        i = 0
+        while i < 5 and size >= base:
+            size /= base
+            i += 1
+        return f'{size:.2f} {units[i]}{unit}'
```

Comment:
I think our API should stick to byte counts, and our clients should use something like this method, or maybe even [humanize](https://python-humanize.readthedocs.io/en/latest/filesize/).

Reply:
This is only used for convenient template rendering, not the API. Is there an alternative to this that's readily available?

Reply:
I misunderstood, ignore me :).